initializr-actuator/pom.xml

```diff
@@ -68,7 +68,6 @@
 		<dependency>
 			<groupId>io.spring.initializr</groupId>
 			<artifactId>initializr-web</artifactId>
-			<scope>test</scope>
```

Comment:
I understand why that's necessary but I don't like the idea of having a link between those two only to be able to order auto-configurations. Perhaps we should consider using an FQN for now (arguably, that auto-configuration shouldn't be in `initializr-web` either. It is there because there isn't a "core" concept atm).